build.gradle

```diff
@@ -17,21 +19,45 @@ def _arch = 'x86_64'
 def _os = 'LINUX'
 def _release = 1
 def _sourcePaths=[]
+def apiVersion = ''
+
+def getCurrentTimestamp(){
+  new Date().format("MM-dd-yyyy_hh-mm")
+}
+
+def updateVersionProperties() {
+    def versionFile = file('src/main/resources/version.properties')
+    String versionProp = versionFile.getText('UTF-8')
+    println "updating version to '${version}' in ${versionFile}"
+    versionProp = versionProp.replaceAll(/version\.major=.*/, "version.major=${version}")
+    versionFile.write(versionProp, 'UTF-8')
+    println "updating docker tag to '${dockerTag}' in ${versionFile}"
+    versionProp = versionProp.replaceAll(/version\.tag=.*/, "version.tag=${dockerTag}")
+    versionFile.write(versionProp, 'UTF-8')
+    println "updating build number to '${build}' in ${versionFile}"
+    versionProp = versionProp.replaceAll(/version\.build=.*/, "version.build=${build}")
```

Comment:
remove the build line.  The version will include the build number

Reply:
gradle.properties

Reply:
Done